src/lib/y2storage/proposal/phys_vol_strategies/base.rb

```diff
@@ -127,12 +127,15 @@ def potential_partition_size(partition, space)
           @potential_part_sizes[partition] ||= space.usable_extra_size + partition.min_size
         end
 
-        # Planned partition that will hold an LVM PV
+        # Planned partition that will hold an LVM PV of the planned physical
+        # volume
```

Comment:
"of the planned volume group", isn't it?

Reply:
Ups, fixed.